--- rxstack.init.d.in	2001-10-31 11:38:40.000000000 +0100
+++ rxstack.init.d.in-shad	2006-05-11 02:03:17.618421208 +0200
@@ -15,7 +15,6 @@
   start)
 	echo -n "Starting Regina Stack: "
 	daemon @prefix@/bin/rxstack -d
-	echo
 	[ $RETVAL -eq 0 ] && touch /var/lock/subsys/rxstack
 	;;
   stop)
@@ -23,7 +22,6 @@
 	killproc @prefix@/bin/rxstack -2
 	RETVAL=$?
 	[ $RETVAL -eq 0 ] && rm -f /var/lock/subsys/rxstack
-	echo
 	;;
   restart|reload)
 	$0 stop
